pkg/controller/kogitojobsservice/kogitojobsservice_controller.go

```diff
@@ -143,8 +144,13 @@ const (
 	maxIntervalLimitRetryEnvKey       = "MAX_INTERVAL_LIMIT_RETRY"
 	backOffRetryDefaultValue          = 1000
 	maxIntervalLimitRetryDefaultValue = 60000
+	kafkaTopicNameJobsEvents          = "kogito-job-service-job-status-events"
```

Comment:
Are you sure about this name ?
Shouldn't we rely on that topic name ?
https://github.com/kiegroup/kogito-apps/blob/7a53dc2f6e2e15e6dcc2ed194e043fc29aee1226/data-index/data-index-service/src/main/resources/application.properties#L42
Also present here: https://github.com/kiegroup/kogito-cloud-operator/blob/c3a80921e3271ea5bf0c28ee3cfb297610bc84c5/pkg/controller/kogitodataindex/kogitodataindex_controller.go#L166
Or do I miss something ?

Reply:
This is the name presented in the properties, this name is used to set the env var:
https://github.com/kiegroup/kogito-apps/blob/master/jobs-service/src/main/resources/application.properties#L74

Reply:
In the Data Index the name differ:
https://github.com/kiegroup/kogito-apps/blob/master/data-index/data-index-service/src/main/resources/application.properties#L42

Reply:
ok. my bad. indeed did not look into the correct app ...